sentry/src/main/java/io/sentry/Scopes.java

```diff
@@ -602,12 +593,28 @@ public ISentryLifecycleToken pushScope() {
           .log(SentryLevel.WARNING, "Instance is disabled and this 'pushScope' call is a no-op.");
       return NoOpScopesStorage.NoOpScopesLifecycleToken.getInstance();
     } else {
-      Scopes scopes = this.forkedCurrentScope("pushScope");
+      final @NotNull IScopes scopes = this.forkedCurrentScope("pushScope");
```

Comment:
Comment on method regarding the return of a lifecycle token can be removed

Reply:
Already removed in a follow up PR